pkg/api/types.go

```diff
@@ -513,7 +513,8 @@ type EnvVar struct {
 	// Required: This must be a C_IDENTIFIER.
 	Name string `json:"name"`
 	// Optional: no more than one of the following may be specified.
-	// Optional: Defaults to "".
+	// Optional: Defaults to ""; variable references ("$(VAR_NAME)") are
```

Comment:
@derekwaynecarr if you like the copy here and in the v1 types.go file, I will change the other API versions.